ui/src/textbox.tsx

```diff
@@ -65,6 +73,8 @@ export interface Textbox {
   tooltip?: S
   /** True if the text may be checked for spelling errors. Defaults to True. */
   spellcheck?: B
+  /** Keyboard to be shown on mobile devices. Defaults to 'text'. */
+  keyboard?: 'text' | 'number' | 'tel'
```

Comment:
Let's go for `type` to keep consistency since we already use it for other widgets as well.